NEWS

```diff
@@ -1,6 +1,11 @@
 shiny-server 1.5.0
 --------------------------------------------------------------------------------
-* Add auto-reconnect capabilities.
+* Add auto-reconnect capabilities. Can be disabled via `disable_reconnect`
```

Comment:
This wording is awkward. Am I disabling reconnecting? Or disabling the disablement of reconnecting?